### ISSUE TITLE
How to shuffle my training data?

### ISSUE THREAD

**Issue by YaoMufeng** [open-mmlab/mmsegmentation#63]
I want to shuffle my data when training.
In pytorch I just add "shuffle=True" in TrainDataloader() to shuffle my data.
Is there any convenient way to do this in mmsegmentation?

**Comment by YaoMufeng** [comment]
I found that I can do this by  modify buildDataLoader in ./mmseg/datasets/builder.py
